core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorBodyDirective.js

```diff
@@ -28,11 +28,27 @@ oppia.directive('simpleEditorBody', [
         '$scope', 'SimpleEditorManagerService',
         'explorationSaveService', 'explorationRightsService',
         'explorationWarningsService', 'QuestionIdService',
-        function($scope, SimpleEditorManagerService,
-            explorationSaveService, explorationRightsService,
-            explorationWarningsService, QuestionIdService) {
+        'StatesToQuestionsService',
+        function($scope, SimpleEditorManagerService, explorationSaveService,
```

Comment:
Let's have the line breaks here match the line breaks above, so that it's easier to compare both lists.